src/main/kotlin/blackjack/domain/betting/BettingResult.kt

```diff
@@ -0,0 +1,37 @@
+package blackjack.domain.betting
+
+import blackjack.domain.participant.Player
+import blackjack.domain.result.BlackjackResult
+import blackjack.domain.result.ResultType
+
+class BettingResult
+private constructor(private val bettingResult: Map<Player, Int>) {
+    fun getPlayerEarningMoney(player: Player) = bettingResult[player]
+    fun getDealerEarningMoney(): Int = -bettingResult.values.sum()
+
+    companion object {
+        private const val BLACKJACK_EARNING_RATE = 1.5
+        private const val TIE_EARNING_RATE = 1.0
+        private const val LOSE_EARNING_RATE = -1.0
+
+        fun of(players: List<Player>, gameResult: BlackjackResult): BettingResult {
+            val results = mutableMapOf<Player, Int>()
+
+            players.forEach {
```

Comment:
```suggestion
            players.forEach { player ->
```
람다에서 기본적으로 제공하는 `it`을 활용해도 좋지만, 가급적이면 변수가 어떤 역할을 하는지 정의하여 사용하는 것을 권장해요.